acquisitions/src/main/resources/thunderjet/mod-orders/reusable/close-order-remove-lines.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Close order and remove composite PO lines
```

Comment:
Can also be annotated with @ignore indicator

Reply:
Added to mod-orders reusables